packages/block-library/src/file/edit.js

```diff
@@ -197,7 +197,7 @@ class FileEdit extends Component {
 							value={ fileName }
 							placeholder={ __( 'Write file name…' ) }
 							keepPlaceholderOnFocus
-							formattingControls={ [] } // disable controls
+							disallowFormats // disable controls
```

Comment:
For something like `fileName` which is a plain string, I wonder why we're using `RichText` at all.

Reply:
Me too. I assume the styles fit better. There's nothing wrong with disallowing all formats, RichText has other benefits like better writing flow adjustments, possible integrations with plugins that do spell checking etc.

I'm wondering if it might be worth having a `PlainText` version that is not a `textarea`, but `contenteditable` and `RichText` under the hood without the formatting.

Reply:
Agree in principle, but wouldn't that be really expensive for what's needed? 🙂